dafny/lcf/README.md

```diff
@@ -0,0 +1,20 @@
+## Project Overview
+
+The Veri-Datalog project is a verified Datalog+ engine. It proves the correctness of Datalog+ programs by constructing proof trees from their trace output. Datalog+ is a research defined language that is similar to Datalog but with more feature support; both are subsets of Prolog.
+
+## Contributions by Rory Brennan-Jones
```

Comment:
Perhaps remove this heading? We know it was your work, but I don't think the README needs to say that.

Reply:
Done